src/app.py

```diff
@@ -1,93 +1,223 @@
-    "Our goal here at Operation Code is to get veterans and their families started on the path to a career in programming. "
-    "We do that through providing you with scholarships, mentoring, career development opportunities, conference tickets, and more!\n\n"
-    "You're currently in Slack, a chat application that serves as the hub of Operation Code. "
-    "If you're currently visiting us via your browser, Slack provides a stand alone program to make staying in touch even more convenient. "
-    "You can download it <https://slack.com/downloads|here.>\n\n"
-    "Want to make your first change to a program right now? "
-    "All active Operation Code Projects are located on our source control repository. "
-    "Our projects can be viewed on <https://github.com/OperationCode/START_HERE|Github.>")
-
-PROXY = PROXY if PROXY else None
+PROXY = config('PROXY', default=None)
+
+TOKEN = config('PERSONAL_APP_TOKEN')
+COMMUNITY_CHANNEL = config('PERSONAL_PRIVATE_CHANNEL')
+
+# TOKEN = config('OPCODE_APP_TOKEN')
+# COMMUNITY_CHANNEL = config('OPCODE_REWRITE_CHANNEL')
+# PROJECTS_CHANNEL = config('OPCODE_OC_PROJECTS_CHANNEL')
+# COMMUNITY_CHANNEL = config('OPCODE_COMMUNITY_ID')
+# COMMUNITY_CHANNEL = config('OPCODE_BOT_TESTING_CHANNEL')
+
+"""Airtable configs"""
+AIRTABLE_BASE_KEY = config('PERSONAL_AIRTABLE_BASE_KEY')
+AIRTABLE_API_KEY = config('PERSONAL_AIRTABLE_TOKEN')
+AIRTABLE_TABLE_NAME = 'Mentor Request'
+
 slack_client = SlackClient(TOKEN, proxies=PROXY)
 
 
-def build_message(message_template, **kwargs):
+# TODO: Do something with all of the return values here
+
+def build_message(message_template: str, **kwargs: dict) -> str:
     return message_template.format(**kwargs)
 
 
-def event_handler(event_dict):
+def event_handler(event_dict: dict) -> None:
+    """
+    Handles routing all of the received subscribed events to the correct method
+    :param event_dict:
+    """
     all_event_logger.info(event_dict)
     if event_dict['type'] == 'team_join':
         new_event_logger.info('New member event recieved')
         new_member(event_dict)
 
-    if event_dict['type'] == 'presence_change':
-        all_event_logger.info('User {} changed state to {}'.format(user_name_from_id(event_dict['user']), event_dict['presence']))
-
-    # can be used for development to trigger the event instead of the team_join
-    if event_dict['type'] == 'message' and 'user' in event_dict.keys():
-
-        # Will need to be removed.  Currently for testing
-        logger.info('Message event')
-    if event_dict['type'] == 'message' and 'user' in event_dict.keys() and event_dict['text'] == 'test4611':
+    """ Trigger for testing team_join event """
+    if event_dict['type'] == 'message' and 'user' in event_dict.keys() and event_dict['text'] == 'testgreet':
         event_dict['user'] = {'id': event_dict['user']}
         new_member(event_dict)
 
 
+def help_menu_interaction(data: dict) -> None:
```

Comment:
Just a general comment. I think that naming this file `app.py` is not correct now that we have so many moving parts. Usually that's what you name the flask handler. We should also start segmenting our files so that When we look at a file it's easier to understand what's going on. 

I think a document that charts out the bot's flow from a new member event to the part where they stop interacting with the bot would be helpful.